elasticsearch/es_util

```diff
@@ -0,0 +1,19 @@
+#/bin/bash -x
+
+# Utility to grab documents from the ES
+# instance using the admin certs and keys
+
+QUERY=${QUERY:-""} 
+INDEX=${INDEX:-"project.*"}
+SIZE=${SIZE:-10}
+SORT=${SORT:-"@timestamp:desc"}
+
+BASE=https://logging-es:9200
```

Comment:
`BASE=${BASE:-https://localhost:9200}`
since you are running this inside the same pod that ES is running, you can use `localhost`.